presto-main/src/test/java/com/facebook/presto/operator/scalar/TestColorFunctions.java

```diff
@@ -130,6 +130,7 @@ public void testRenderDouble()
             throws Exception
     {
         assertEquals(render(1234.5678, color(toSlice("red"))), toSlice("\u001b[38;5;1m1234.5678\u001b[0m"));
+        assertEquals(render(1234.5678f, color(toSlice("red"))), toSlice("\u001b[38;5;1m1234.5677490234375\u001b[0m"));
```

Comment:
I'd at least express this in test, e.g. by building string of float casted to double.
E.g.: `toSlice(format("\u001b[38;5;1m%f\u001b[0m", (double) 1234.5678f));`.
Then it's clear we're relying in FLOAT -> DOUBLE coercion here.
What you think?